contents/copy/word-list.html

```diff
@@ -372,6 +372,10 @@ <h3>settings</h3>
   <p>
     Use “settings” (instead of “options” or “preferences”) for Firefox for mobile and Firefox Focus to conform with the platforms; see also, <a href="#preferences">preferences</a>.
   </p>
+  <h3>Sign in</h3>
+  <p>
+    Sign in Use "sign in to Firefox" or "log in to Firefox" when you are instructing a user to enter site credentials. Do not use "sign into" or "log into".
```

Comment:
why does it start with `Sign in` again?

Reply:
Good catch! @aminalhazwani can you fix it?

Reply:
@brassy- 👍